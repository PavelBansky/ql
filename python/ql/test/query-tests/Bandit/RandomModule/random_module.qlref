Bandit/random_module.ql