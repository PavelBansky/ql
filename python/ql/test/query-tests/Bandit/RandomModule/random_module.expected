| Test.py:5:7:5:21 | Attribute() | Standard pseudo-random generators are not suitable for security/cryptographic purposes. |
| Test.py:6:7:6:24 | Attribute() | Standard pseudo-random generators are not suitable for security/cryptographic purposes. |
| Test.py:7:7:7:22 | Attribute() | Standard pseudo-random generators are not suitable for security/cryptographic purposes. |
| Test.py:8:7:8:21 | Attribute() | Standard pseudo-random generators are not suitable for security/cryptographic purposes. |
| Test.py:9:7:9:22 | Attribute() | Standard pseudo-random generators are not suitable for security/cryptographic purposes. |
| Test.py:10:7:10:25 | Attribute() | Standard pseudo-random generators are not suitable for security/cryptographic purposes. |
